src/lib/sectionList/useSectionListSortOrder.ts

```diff
@@ -78,5 +78,5 @@ export const useSectionListSortOrder = () => {
 export const useSortOrderQueryParams = () => {
     const [sortOrder] = useSectionListSortOrder()
 
-    return sortOrder ? formatSortOrderToString(sortOrder) : undefined
+    return sortOrder ? `${sortOrder[0]}:i${sortOrder[1]}` : undefined
```

Comment:
I think I would prefer to add `idesc` and `iasc` to `sortOrderDirections` "enum". And then handle "translating" directions in ColumnHeaderSortable?

Reply:
I think so. 
yes i though the url makes more sense with the asc and desc, which is also what the table uses, Sure, I can move it in there, probably having a variable called caseSensitive helps the understanding of what the i does so happy w that